### PR TITLE
Feature/polymin

### PR DESCRIPTION
**Pull request by AndSak**

### CODE REVIEW

File: Polindrom/script.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+function palindrome(str) {
+
```
Comment (AndrZakhara) [inline Polindrom/script.js:2]: image

File: Polindrom/script.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+function palindrome(str) {
```
Comment (AndrZakhara) [inline Polindrom/script.js:1]: image

Comment (AndrZakhara) [reply Polindrom/script.js:1]: rename the branch and add numbers for regex please

**Review by AndrZakhara — COMMENTED** [review]
rename the branch and add numbers for regex please

File: Polindrom/script.js (hunk 2 above)
Comment (AndSak) [reply Polindrom/script.js:1]: I have noticed it yesterday,  i thought to fix name`s branch this morning, but you have seen the wrong branch early then i repair it

File: Polindrom/script.js (hunk 1 above)
Comment (AndSak) [reply Polindrom/script.js:2]: in my opinion that is bug. 
 here is more difficult the expression:
**_palindrome("My age is 0, 0 si ega ym.") should return true._**
and everything works

**Pull request closed without merging**